vault-cfn-template/Vault-template.json

```diff
@@ -0,0 +1,285 @@
+{
```

Comment:
Instead of a top-level directory, can you move this to `quick-deploy/cloudformation/vault/`?